packages/js/components/src/experimental-select-control/menu.tsx

```diff
@@ -29,7 +70,7 @@ export const Menu = ( { children, getMenuProps, isOpen }: MenuProps ) => {
 				}
 			) }
 		>
-			{ isOpen && children }
+			{ isOpen && childrenPopoverMarkup }
```

Comment:
This is semantically incorrect. An `ul` element must have `li` elements as its direct children. See `Permitted content` section from [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul). The `ul` element should be at line 44 replacing the `div` and `children` must be `li` elements.

Reply:
Thanks, yeah I just fixed this as part of: https://github.com/woocommerce/woocommerce/pull/34967/commits/043975f49ca8e259a91e9bcfc4adcdeaec63cf58

Reply:
It was causing some styling glitches to, as the screenshot above shows well.